contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.2.0--2.3.0.sql

```diff
@@ -4174,46 +4174,6 @@ CALL sys.babelfish_update_collation_to_default('sys', 'objects', 'type_desc');
 CALL sys.babelfish_update_collation_to_default('sys', 'sysobjects', 'xtype');
 CALL sys.babelfish_update_collation_to_default('sys', 'sysobjects', 'type');
 
--- TODO: BABEL-3127
-CREATE OR REPLACE VIEW sys.all_sql_modules_internal AS
```

Comment:
Please do not remove anything from previous upgrade scripts. It will cause more unpredictable problems. Also, you are modifying old schedule files, that is already a bad sign.